common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -28,17 +27,9 @@ run() {
 
 killall_program() {
     PROGRAM_NAME=$1
-    if [ "$#" -eq 2 ]; then
-        KILL_SIG=$2
-    else
-        KILL_SIG=$SIG_KILL
-    fi
-    for PID in $(pgrep -f "$PROGRAM_NAME"); do
-        if [ "$(basename "$(readlink /proc/"$PID"/exe)")" = "$PROGRAM_NAME" ]; then
-            echo "kill $KILL_SIG $PID $PROGRAM_NAME";
-            kill $KILL_SIG "$PID" > /dev/null 2>&1;
-        fi
-    done
+    KILL_SIG=${2-SIGKILL}
```

Comment:
> Might be worth it to use ${2:-SIGKILL} instead (so that if KILL_SIG is set but null, SIGKILL is still used
That should be, if `$2` is set but null. So when it's called as `killall_program foo ""`. Unlikely :-) But better anyway.